src/Nox.Solution/NoxSolutionBuilder.cs

```diff
@@ -189,11 +189,24 @@ private NoxSolution ResolveAndLoadConfiguration()
             // Validate and deserialize
             var config = NoxSchemaValidator.Deserialize<NoxSolution>(yaml);
 
+            EvaluateCalculatedProperties(config);
+
             config.RootYamlFile = _yamlFilePath;
 
             return config;
         }
 
+        private void EvaluateCalculatedProperties(NoxSolution config)
```

Comment:
I would prefer to be explicit ComputeIsOwnedEntityForAll()

Reply:
For now yes, but I just wanted to mark this as a place where all similar calculated properties are evaluated